src/xcode/ENA/ENA/Source/Client/__tests__/Mocks/CachingHTTPClientMock.swift

```diff
@@ -41,6 +41,13 @@ final class CachingHTTPClientMock: CachingHTTPClient {
 		return configMetadata
 	}()
 
+	static let statiVaccinationValueSets: SAP_Internal_Dgc_ValueSets = {
+		let bundle = Bundle(for: CachingHTTPClientMock.self)
+		guard let configMetadata = try? SAP_Internal_Dgc_ValueSets(jsonString: "{\"vp\":{\"items\":[{\"key\":\"1119349007\",\"displayText\":\"SARS-CoV-2 mRNA vaccine\"}]},\"mp\":{\"items\":[{\"key\":\"EU/1/20/1528\",\"displayText\":\"Comirnaty\"}]},\"ma\":{\"items\":[{\"key\":\"ORG-100001699\",\"displayText\":\"AstraZeneca AB\"}]}}") else {
```

Comment:
can this string be stored in a separate file?

Reply:
it is not that big and only for this test

Reply:
Okay.